### DISCUSSION TITLE
[Feature Request]: 能否恢复 1.8.1版本之前的那个记住刮削内容啊

### DISCUSSION THREAD

**Discussion by yokunnyang** [jxxghp/MoviePilot#3451] [讨论]
### 当前程序版本

1.8.5

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

请求，恢复1.8.1之前这个，历史记录手动刮削整理时上次填写的信息，如图，让他下次点文件整理的时候能记住我图上那个设置（就是上一次我成功手动刮削填写的记录），拜托了

![微信截图_20240430002115](https://github.com/jxxghp/MoviePilot/assets/133833844/272619b6-1499-411f-8748-79e6be0e0292)

**Comment by Katou-Kouseki** [comment]
这个操作起来确实感觉顺手不少

**Comment by jxxghp** [comment]
用集数定位批量处理，为啥要一个个去整理？
